cfme/tests/infrastructure/test_vmware_provider.py

```diff
@@ -66,9 +71,33 @@ def test_appliance_scsi_control_vmware():
         casecomponent: Appliance
         caseimportance: critical
         initialEstimate: 1/4h
-    #TODO: yet to test this, once done, I will add steps. Test was not written by me originally.
     """
-    pass
+    url = ("{baseurl}/{series}/{ver}/cfme-vsphere-paravirtual-{ver}-1.x86_64.vsphere.ova"
+        .format(baseurl=conf.cfme_data['basic_info']['cfme_images_url']['baseurl'],
+            series=appliance.version.series(), ver=appliance.version))
+    logger.info("Downloading ova file for parvirtual vsphere scsi controller test from %s" % url)
+    ova_file = urllib.urlopen(url)
+    filename = os.path.basename(url)
+    with open(filename, 'wb') as output:
+        output.write(ova_file.read())
+
+    @request.addfinalizer
+    def _cleanup():
+        if os.path.exists(filename):
+            os.remove(filename)
+    tar = tarfile.open(filename)
```

Comment:
Please use the context manager here when reading the tar archive, so that it gets closed.